libcontainer/utils/utils_unix.go

```diff
@@ -67,3 +70,22 @@ func NewSockPair(name string) (parent *os.File, child *os.File, err error) {
 	}
 	return os.NewFile(uintptr(fds[1]), name+"-p"), os.NewFile(uintptr(fds[0]), name+"-c"), nil
 }
+
+const (
+	IoprioWhoPgrp = 1
+)
+
+func SetIOPriority(ioprio *configs.IOPriority) error {
+	class, ok := configs.IOPrioClassMapping[ioprio.Class]
+	if !ok {
+		return fmt.Errorf("invalid io priority class: %s", ioprio.Class)
+	}
+
+	// Combine class and priority into a single value
+	iop := (class << 13) | ioprio.Priority
```

Comment:
Could you please put the official reference doc here? I saw a different method in https://github.com/torvalds/linux/blob/v5.18/include/uapi/linux/ioprio.h#L5-L17

Reply:
Do you know why linux do a more complex operation in `IOPRIO_PRIO_VALUE`, although the result is the same as you here `(class << 13) | ioprio.Priority`.